filebeat/docs/inputs/input-syslog.asciidoc

```diff
@@ -0,0 +1,39 @@
+:type: syslog
+
+[id="{beatname_lc}-input-{type}"]
+=== Syslog input
+
+++++
+<titleabbrev>Syslog/titleabbrev>
+++++
+
+Use the `syslog` input to read events over TCP or UDP.
+
+Example configurations:
+
+["source","yaml",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- type: syslog
+  protocol.udp:
+    host: "localhost:9000"
+----
+
+["source","yaml",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- type: syslog
+  protocol.tcp:
+    host: "localhost:9000"
+----
+
+==== Configuration options
+
+The `syslog` input supports the following configuration options plus the
+<<{beatname_lc}-input-{type}-common-options>> described later.
+
+
+[id="{beatname_lc}-input-{type}-common-options"]
```

Comment:
Yup, I'd do something similar to what I've done here:

https://raw.githubusercontent.com/elastic/beats/master/filebeat/docs/inputs/input-common-file-options.asciidoc

So maybe you would have inputs/input-common-tcp-options.asciidoc and input-common-udp-options.asciidoc. Make sure that you add a comment to the top of the file explaining how the options are shared (see input-common-file-options.asciidoc for an example). 

Of course, you'll need to use attributes to resolve the section IDs for each option because the IDs need to be unique across the book. Something like:

[id="{beatname_lc}-input-{type}-option-name"]